lib/hookRunner.js

```diff
@@ -20,7 +20,8 @@ function hookRunner (functions, runner, state, cb) {
 
     const result = runner(functions[i++], state, next)
     if (result && typeof result.then === 'function') {
-      result.then(handleResolve, handleReject)
+      result
```

Comment:
Given that there is no `.catch`, I wouldn't add the newline on this one.

Reply:
done